Makefile

```diff
@@ -13,7 +13,8 @@ build: $(THEME)
 
 .PHONY: update
 update: $(THEME)
```

Comment:
I think we can drop this task now. Maybe we should also add a clean task and put the url into a variable than you can overwrite it locally or even later on for different branches within drone with an environment variable.